dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1111,10 +1138,12 @@ Spdp::handle_participant_crypto_tokens(const DDS::Security::ParticipantVolatileM
   // If discovery hasn't initialized / validated this participant yet, ignore volatile message
   DiscoveredParticipantIter iter = participants_.find(src_participant);
   if (iter == participants_.end()) {
-    ACE_DEBUG((LM_WARNING,
-      ACE_TEXT("(%P|%t) Spdp::handle_participant_crypto_tokens() - ")
-      ACE_TEXT("received tokens for undiscovered participant %C. Ignoring.\n"),
-               std::string(DCPS::GuidConverter(src_participant)).c_str()));
+    if (OpenDDS::DCPS::security_debug.auth_warn) {
+      ACE_ERROR((LM_WARNING,
+        ACE_TEXT("(%P|%t) {auth_warn} Spdp::handle_participant_crypto_tokens() - ")
+        ACE_TEXT("received tokens for undiscovered participant %C. Ignoring.\n"),
+        std::string(DCPS::GuidConverter(src_participant)).c_str()));
```

Comment:
Should this be an `OPENDDS_STRING`?

Reply:
It doesn't need to be since there's no safety profile support here (yet), but it doesn't hurt to do that now